subsys/bluetooth/host/l2cap.c

```diff
@@ -1122,6 +1124,12 @@ static void le_ecred_conn_req(struct bt_l2cap *l2cap, uint8_t ident,
 	BT_DBG("psm 0x%02x mtu %u mps %u credits %u", psm, mtu, mps, credits);
 
 	if (mtu < L2CAP_ECRED_MIN_MTU || mps < L2CAP_ECRED_MIN_MTU) {
+		BT_ERR("Invalid ecred conn req params");
+		result = BT_L2CAP_LE_ERR_INVALID_PARAMS;
```

Comment:
I don't see tests for this MTU check for L2CAP without ECFC. Also, Core specification doesn't tell anything about these errors, only lists their codes. Maybe this can be added to `le_conn_req`, but it shouldn't affect any tests.

Reply:
I was referring to this: https://github.com/zephyrproject-rtos/zephyr/blob/715b68d389b9cdb75208a48663035c48a34828b0/subsys/bluetooth/host/l2cap.c#L1044L1047

It is not related to your changes here, I just find it weird.

Do you know if there is any tests that verifies that behavior?

Reply:
Yes, that’s what I meant. I haven’t seen any tests of this, for not enhanced credit flow control